src/app.py

```diff
@@ -1,637 +1,12 @@
-import json
-import os
-from datetime import datetime
-
-import boto3
-import github_api_toolkit
-import pandas as pd
-import plotly.graph_objects as go
 import streamlit as st
-from botocore.exceptions import ClientError
-from plotly.subplots import make_subplots
-
-# GitHub Organisation
-org = os.getenv("GITHUB_ORG")
-
-# GitHub App Client ID
-client_id = os.getenv("GITHUB_APP_CLIENT_ID")
-
-# AWS Secret Manager Secret Name for the .pem file
-secret_name = os.getenv("AWS_SECRET_NAME")
-secret_reigon = os.getenv("AWS_DEFAULT_REGION")
-
-account = os.getenv("AWS_ACCOUNT_NAME")
-
-# AWS Bucket Path
-bucket_name = f"{account}-copilot-usage-dashboard"
-object_name = "historic_usage_data.json"
-
-
-@st.cache_data
-def get_pem_from_secret_manager(_session: boto3.Session, secret_name: str, region_name: str) -> str:
-    """Gets the .pem file contents from AWS Secret Manager
-
-    Args:
-        session (boto3.Session): A boto3 session (Led with an underscore so st.cahce_data doesn't hash it)
-        secret_name (str): The name of the secret in AWS Secret Manager
-        region_name (str): The region where the secret is stored
-    Returns:
-        str: The contents of the .pem file
-    """
-    secret_manager = session.client("secretsmanager", region_name=region_name)
-    return secret_manager.get_secret_value(SecretId=secret_name)["SecretString"]
-
-
-# Initialise a boto3 session
-session = boto3.Session()
 
 st.set_page_config(
     page_title="CoPilot Usage Dashboard",
     page_icon="./src/branding/ONS-symbol_digital.svg",
     layout="wide",
 )
+create_page = st.Page("./pages/org_usage.py", title="Organization Usage", icon=":material/groups:")
+delete_page = st.Page("./pages/team_usage.py", title="Team Usage", icon=":material/group:")
```

Comment:
Why are the variable names create_page and delete_page ?  Couldn't it be org_page and team_page?

Reply:
Changed.